release-notes/6.0/preview/api-diff/preview7/.Net/6.0.0-preview7_System.Threading.md

```diff
@@ -0,0 +1,12 @@
+# System.Threading
+
+``` diff
+ namespace System.Threading {
++    public sealed class PeriodicTimer : IDisposable {
```

Comment:
👍🏾

Reply:
Nope.  I proposed it and it was shot down.